azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/BatchAccount.java

```diff
@@ -81,6 +82,34 @@
      */
     Map<String, Application> applications();
 
+    /**
+     * @return pools in this Batch account, indexed by name
+     */
+    Map<String, Pool> pools();
+
+    /**
+     * @return dedicatedCoreQuotaPerVMFamilyEnforced value indicating
+     * whether the core quota for the Batch Account is enforced
```

Comment:
`dedicatedCoreQuotaPerVMFamilyEnforced` is not necessary here

`Virtual Machine` ==> lower case `virtual machine`

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L2034
should we change to lower case?